src/modules/client/components/planning/Planning.vue

```diff
@@ -270,7 +271,8 @@ export default {
     async drop (toDay, target) {
       try {
         if (target.type === SECTOR) { // Unassign event
-          if (this.draggedObject.sector === target._id && (!this.draggedObject.auxiliary || !this.draggedObject.auxiliary._id) &&
+          if (this.draggedObject.sector === target._id &&
+            (!this.draggedObject.auxiliary || !this.draggedObject.auxiliary._id) &&
```

Comment:
tu peux mettre !get(this.draggedObject, 'auxiliary._id')
et aussi peut etre nommer les conditions parce que la c'est un peu dur a lire
```
const dropToSameSector = this.draggedObject.sector === target._id && !get(this.draggedObject, 'auxiliary._id')
const dropToSameDay = toDay.isSame(this.draggedObject.startDate, 'd')
if (dropToSameSector && dropToSameDay) return;
```